python-sdk/tests_integration/sql/operators/test_merge.py

```diff
@@ -452,8 +475,9 @@ def get_result_sql_single(conn_id):
         (base_database_class, "gcp_conn"),
         (base_database_class, "snowflake_conn"),
         (delta_database_class, "databricks_conn"),
+        (base_database_class, "duckdb_conn"),
     ],
-    ids=["sqlite", "bigquery", "snowflake", "databricks"],
+    ids=["sqlite", "bigquery", "snowflake", "databricks", "duckdb"],
 )
```

Comment:
Can we extend duck db for these tests? - test_merge_with_the_same_schema, run_merge, test_merge_with_different_schemas

Reply:
Sure.

Reply:
@utkarsharma2 added, please check: https://github.com/astronomer/astro-sdk/pull/1695/commits/d8bbd2c2314eaf5bb23df2ff83a7fbf9756be4bf

Reply:
Can we mention a comment in the code next to test cases that won't work for duck DB?